app/assets/stylesheets/scaffolds.scss

```diff
@@ -57,6 +59,7 @@ body {
   &.stories-search,
   &.podcast_episodes-index,
   &.reading_list_items-index,
+  &.history-index,
```

Comment:
added history index to here, we should probably use a common class for all of these containers, instead of listing them explicitly, it took me a while to understand why there was no margin :D